packages/dcos-integration-test/extra/conftest.py

```diff
@@ -41,6 +42,31 @@ def pytest_collection_modifyitems(session, config, items):
     items[:] = new_items + last_items
 
 
+def pytest_addoption(parser):
+    parser.addoption('--resiliency', action='store_true')
```

Comment:
/me really wishes we could do this as a test "suite" rather than doing a special marker here. Really easy to forget the marker